frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/BatchAggregator.java

```diff
@@ -90,6 +74,7 @@ public BaseModelRequest getRequest() throws InterruptedException {
         ModelInferenceRequest req = new ModelInferenceRequest(model.getModelName());
         for (Job j : jobs.values()) {
             req.addRequestBatches(j.getPayload());
+            req.setSynchronous(true);
```

Comment:
Default is true, redundant set

Reply:
True. I had added it to avoid any async messages triggering a failure. But we could adopt one of the two approaches.
1. Add this back when we have actual sync messages in the future to the backend.
2. Make backend worker always return a message (making it always a sync)

I will remove this for now, as its not needed.